Teamleader.php

```diff
@@ -414,6 +414,28 @@ public function crmUpdateContact(
     }
 
     /**
+     * Delete a contact
+     *
+     * @todo    N/A
+     *
+     * @param         $contact can be either an object of type "Contact" or a contact ID
```

Comment:
Actually, I think it would be better to support `@param int|string|Contact`.

A string can easily be converted to an int with `$fields['contact_id'] = (int) $contact`, analogous to other methods.